torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -445,6 +445,33 @@ void PyTorchModelLoader::loadMaxPool2d(const torch::jit::Node *ptNode) {
   addGlowNodeValue(outputs[0], output);
 }
 
+void PyTorchModelLoader::loadReshape(const torch::jit::Node *ptNode) {
```

Comment:
General question.

Could any of the load methods fail, like some of the params are not supported etc?
In that case I'm wondering why it's all `void`.
cc: @jackm321

Reply:
Good point, I think we need error checking mechanism.

Reply:
Yeah, we will add llvm::Errors like in c2 model loader. It's on the todo list :)